google_fonts/README.md

```diff
@@ -1,14 +1,64 @@
-# google_fonts_test
+# google_fonts
 
-A new Flutter package.
+The GoogleFonts package for Flutter allows you to easily use any of the 960 fonts
+(and their variants) from fonts.google.com in your flutter app.
 
 ## Getting Started
 
-This project is a starting point for a Dart
-[package](https://flutter.dev/developing-packages/),
-a library module containing code that can be shared easily across
-multiple Flutter or Dart projects.
+![](main.gif)
```

Comment:
Can you put in the actual gif here?

![google_fonts](https://user-images.githubusercontent.com/4229329/69070973-1dd08b80-09f7-11ea-89a8-c3e0b17075e8.gif)